### PR TITLE
Replace Loki with Vali

### PR DESCRIPTION
**Pull request by Kristian-ZH**
**How to categorize this PR?**

/kind enhancement
/area logging

**What this PR does / why we need it**:
This PR replaces Loki with Vali

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
@vlvasilev 

**Release note**:

```other operator
Loki is now replaced with Vali: https://github.com/credativ/vali
```

### CODE REVIEW

**Review by istvanballok — COMMENTED** [review]
/lgtm

**Review by vlvasilev — APPROVED** [review]
/lgtm

**Review by vlvasilev — CHANGES_REQUESTED** [review]
except the version postfix all is lgtm

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v0.49.0-dev
\ No newline at end of file
+v0.49.0-dev-vali-3
```
[the remark this answers is not in the document]
Comment (Kristian-ZH) [reply VERSION:1]: my bad, that is a leftover from the local testing

**Pull request closed without merging**